packages/desktop-client/src/components/Titlebar.js

```diff
@@ -238,16 +239,12 @@ function BudgetTitlebar({ globalPrefs, saveGlobalPrefs, localPrefs }) {
                 </ButtonWithLoading>
               </P>
               <P isLast={true}>
-                {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                <a
-                  href="#"
-                  style={{
-                    color: colors.n4,
-                    fontStyle: 'italic',
-                  }}
+                <ExternalLink
+                  to="https://actualbudget.org/docs/experimental/report-budget"
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏good one! So this was broken for quite some time.. :D

Reply:
There originally wasn’t a doc page for this.